src/character.cpp

```diff
@@ -3944,3 +3944,17 @@ void Character::set_fac_id( const std::string &my_fac_id )
 {
     fac_id = string_id<faction>( my_fac_id );
 }
+
+std::string get_stat_name( Character::stat Stat )
+{
+    switch( Stat ) {
+        // *INDENT-OFF*
```

Comment:
It's the end of the file, so not a big deal, but you forgot the `INDENT-ON`.

Reply:
So many mistake today U_U

Reply:
Oh yeah but I put it back in #34177